src/backend/aspen/workflows/pangolin/tests/test_pangolin_workflow.py

```diff
@@ -42,3 +59,35 @@ def test_pangolin_export(mocker, session, postgres_database):
     with open("test.fa", "r") as fh:
         lines = fh.read()
         assert lines == ">1\nTCGGCG>2\nTCGGCG"
+
+
+def test_pangolin_save(mocker, session, postgres_database):
+
+    create_test_data(session)
+    mock_remote_db_uri(mocker, postgres_database.as_uri())
+
+    pangolin_csv: PosixPath = Path(Path(__file__).parent, "data", "lineage_report.csv")
+
+    runner: CliRunner = CliRunner()
+    result: Result = runner.invoke(
+        save_cli,
+        ["--pangolin-csv", pangolin_csv, "--pangolin-last-updated", "05-03-2021"],
+    )
+    assert result.exit_code == 0
+
+    # start new transaction
+    session.close()
+    session.begin()
+
+    for i in range(1, 3):
+        pathogen_genome: UploadedPathogenGenome = (
+            session.query(UploadedPathogenGenome)
+            .filter(UploadedPathogenGenome.entity_id == i)
+            .one()
+        )
```

Comment:
i would not rely on the entity_id being a certain number.  you do know what the public_identifier will be, so you can query on that.

Reply:
i actually don't know what the public_identifier will be because the taxon is set to be entity_id (from when we renamed the sequences in export.py)

Reply:
should i change export script to output public_identifier instead of entity_id?

Reply:
You don't need to, but there is no guarantee that entity_id will be 1-3.  Maybe you should just pull all the entries from the DB, and then compare it against what you expect to find.

Reply:
oh i thought you were talking about the overall logic and not the test, ok will fix

Reply:
ok i changed it